### PR TITLE
profiles: hardening and self-ref fixes

### PR DESCRIPTION
**Pull request by glitsj16**
We have 2 profiles that refer to `firejail` in private-etc (ghostwriter and peek), and one (peek) additionally includes it private-bin. IMO there is never a legitimate reason for this. This PR removes firejail from both profiles.

### CODE REVIEW

**Comment by glitsj16** [comment]
@rusty-snake I noticed the comment in ghostwriter.profile on #2877. If it's not too much trouble, can you take a look/review this from that perspective?

File: etc/profile-a-l/ghostwriter.profile [hunk 1]
```diff
@@ -21,10 +21,17 @@ include disable-programs.inc
 include disable-shell.inc
 include disable-xdg.inc
 
+whitelist ${HOME}/.config/ghostwriter
+whitelist ${HOME}/.local/share/ghostwriter
+whitelist ${DOCUMENTS}
+whitelist ${DOWNLOADS}
+whitelist ${PICTURES}
 whitelist /usr/share/ghostwriter
+whitelist  /usr/share/mathjax
```
Comment (rusty-snake) [inline etc/profile-a-l/ghostwriter.profile:30]:
```suggestion
whitelist  usr/share/mathjax
```

File: etc/profile-a-l/ghostwriter.profile [hunk 2]
```diff
@@ -47,14 +54,15 @@ seccomp !chroot
...
+private-bin affixcompress,analyze,chmorph,cmark,context,gettext,ghostwriter,hunspell,hunzip,hzip,latex,makealias,mktexfmt,munch,multimarkdown,pandoc,pdflatex,pdfroff,prince,unmunch,weasyprint,wkhtmltopdf,wordforms,wordlist2hunspell
 private-cache
 private-dev
 # passwd,login.defs,firejail are a temporary workaround for #2877 and can be removed once it is fixed
-private-etc @tls-ca,@x11,dbus-1,firejail,gconf,host.conf,mime.types,rpc,services,texlive
+private-etc @tls-ca,@x11,dbus-1,gconf,host.conf,mime.types,rpc,services,texlive
 private-tmp
 
 dbus-user filter
+dbus-user.talk ca.desrt.dconf
```
Comment (rusty-snake) [inline etc/profile-a-l/ghostwriter.profile:65]: It is a Qt program. Why alloe dconf?

Comment (glitsj16) [reply etc/profile-a-l/ghostwriter.profile]: Nice catch, thanks! Probably I mixed things up with the peek profile. Anyway, I took it out.

File: etc/profile-a-l/ghostwriter.profile (hunk 1 above)
Comment (rusty-snake) [reply etc/profile-a-l/ghostwriter.profile]: Or I just had the cursor of by one when hitting backspace.

**Comment by netblue30** [comment]
Merged, thanks!

**Pull request merged**